net/sslh/src/opnsense/mvc/app/models/OPNsense/Sslh/Menu/Menu.xml

```diff
@@ -0,0 +1,6 @@
+<menu>
+    <Services>
+        <SSLH VisibleName="SSLH" cssClass="fa fa-lock fa-fw" url="/ui/sslh/settings">
```

Comment:
Even if it looks silly usually we opt for "Sslh" although that looks strange...  a bit like Dnsmasq.

```suggestion
        <SSLH cssClass="fa fa-lock fa-fw" url="/ui/sslh/settings">
```

visible name is already set via xml node.